RenderSystems/GL3Plus/src/OgreGL3PlusHardwareBufferManager.cpp

```diff
@@ -48,7 +48,6 @@ namespace Ogre {
         uint32 free: 1;
     };
 #define SCRATCH_POOL_SIZE 1 * 1024 * 1024
-#define SCRATCH_ALIGNMENT 32
```

Comment:
this is in bits, but OGRE_MALLOC_ALIGN took byte sizes, so this is satisfied on any 32bit+ system.